infrastructure/src/main/java/org/corfudb/infrastructure/AbstractServer.java

```diff
@@ -16,6 +20,11 @@
     @Setter
     volatile boolean shutdown;
 
+    static final ExecutorService sharedExecutor = Executors
```

Comment:
Can we try to separate datapath (sequencer/log unit) thread pool(s) from control path (Layout/Management server) thread pool(s). The control path can run on single or at most 2 threads. Since their queues will be separate they will not be able interfere so much with datapath.